presentation/backend/src/presentation-backend/PresentationManager.ts

```diff
@@ -296,6 +296,12 @@ export interface PresentationManagerProps {
 
   /** @internal */
   addon?: NativePlatformDefinition;
+
+  /*
+   * A function that will be called after receiving diagnostics.
+   * @alpha
```

Comment:
Why is it `@alpha`? Such APIs are not publicly documented. If you really want it to be alpha, don't include in public `NextVersion.md` documentation.

Reply:
Do  you intend to leave other APIs as alpha?

Reply:
Changed all APIs related to diagnostics to `@beta`.